.prow.yaml

```diff
@@ -28,7 +28,7 @@ presubmits:
       containers:
         - image: quay.io/thoth-station/thoth-pytest-py38:v0.12.8
           command:
-             - "/bin/run-pytest"
+             - pytest
```

Comment:
This should not be changed, because it changes the interface how we want pytest to be invoced.

Reply:
Fixed